dotnet/DataSources/ProductReviewRepository.cs

```
@@ -555,14 +582,15 @@ public async Task<string> SuccessfulMigration()
                 
                 var client = _clientFactory.CreateClient();
                 var response = await client.SendAsync(request);
-                string responseContent = await response.Content.ReadAsStringAsync();
-                return responseContent;
+                responseContent = await response.Content.ReadAsStringAsync();
+                
             }
             catch (Exception ex)
             {
-                Console.WriteLine(ex.InnerException.Message);
-                return ex.InnerException.Message;
+                _context.Vtex.Logger.Error("SuccessfulMigration", null, "Error:", ex);
```

we still need to return the error message for catch result